command/connect/envoy/bootstrap_tpl.go

```diff
@@ -96,6 +96,8 @@ type BootstrapTplArgs struct {
 	// Namespace is the Consul Enterprise Namespace of the proxy service instance as
 	// registered with the Consul agent.
 	Namespace string
+
+	EnvoyVersion string
```

Comment:
nit: every other field here has a comment explaining it which I think is a handy place to document things internally even if it's not a public API. There's not a lot to say about EnvoyVersion but it might not be obvious for example why it matters?

Reply:
Absolutely.